user_guide/admin_capabilities.rst

```diff
@@ -0,0 +1,461 @@
+
+|
+
+.. image:: ./img/roles/export.png 
+
+|
+
+This is a general productivity capability that is usually enabled for most users.
+
+
+.. _admin-capabilities-general_import:  
+
+General import/action operations
+--------------------------------
+
+The capability enables common operations for import and export, such as the action to load the error log. It is 
+recommended to set it to *"System"* if at least one of the capabilities *"Export Entities"* and *"Import Entities"* is 
+set to *"System"*. 
+
+
+
+.. _admin-capabilities-import-entities:    
+
+Import Entities
+---------------
+
+If this capability is set to *"System"*, users with the role can import records from a .CSV file to OroCRM, as described 
+in the :ref:`Import and Export Data <user-guide-import>` guide. The :guilabel:`Import` button will 
+appear in the top right corner of the page.
+
+|
+
+.. image:: ./img/roles/import.png 
+
+|
+
+This capability is necessary for users who need to import large sets of data into the system, for example these may be 
+sales representatives or employees responsible for the lead development.
+
+.. _admin-capabilities-jobs:  
+
+Jobs management
+---------------
+
+If this capability is set to *"System"*, users with the role can see jobs started in the system, view their currrent 
+status and their performance log from the *"Job Queue*" and *"Sheduled tasks"* pages. Links to these pages are 
+available in the *"System"* menu.
+
+This capability is usually used by the system administrators.
+
+
+.. _admin-capabilities-mailchimp:  
+
+MailChimp manipulations
+-----------------------
+
+The capability is meaningful if the system has been integrated with Mailchimp. Such integration enables users to utilize 
+MailChimp as an email campaign engine  as described in the guide on 
+:ref:`Integration with MailChimp <user-guide-mc-integration>`. 
+
+When using the functionality, the user can synchronize lists of contacts in OroCRM and MailChimp, use them to create 
+email campaigns in MailChimp and import them to OroCRM – the MailChimp campaign statistics can then be used to analyze 
+the campaign efficiency. All of these operations on the OroCRM side will only be available if the capability has been 
+set to  *"System"*.
+
+The capability is usually enabled for marketing associates and other managers responsible for email campaigns.
+
+
+.. _admin-capabilities-config-entities:  
+
+Manage configurable entities
+----------------------------
+
+Many entities in OroCRM can be configured from the UI, as described in the 
+:ref:`Entities guide <user-guide-entity-management-from-UI>`. So, the system administrator can change the attachments 
+settings, define whether the entity should be displayed on a grid and/or a View page, whether it will be exported 
+to a .CSV file, and define other settings. For some of them it is also possible to add new fields, as described in the 
+:ref:`Entity Fields guide <user-guide-field-management>`. 
```

Comment:
Edit: Many entities in OroCRM can be configured from the UI, as described in the :ref:`Entities guide <user-guide-entity-management-from-UI>`. The system administrator can change the attachments settings, define whether the entity should be displayed on a grid and/or a View page, whether it will be exported to a .CSV file, and define other settings. For some of them, it is also possible to add new fields, as described in the :ref:`Entity Fields guide <user-guide-field-management>`.